.github/workflows/coverity.yml

```diff
@@ -0,0 +1,32 @@
+name: Coverity scan
+on:
+  # run once daily at 00:30 UTC due to
+  # https://scan.coverity.com/faq#frequency
+  schedule:
+    - cron: "30 0 * * *"
+
+jobs:
+  coverity:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        id: dependencies
+        uses: ./.github/actions/install-dependencies
+
+      - name: Configure
+        run: |
+          source contrib/fedora/bashrc_sssd
+          cd contrib/ci/
+          . configure.sh
+          cd ../..
+          reconfig "${CONFIGURE_ARG_LIST[@]}"
+
+      - name: Execute and submit coverity scan
+        uses: vapier/coverity-scan-action@v1.2.0
+        with:
+          email: "sssd-team@redhat.com"
+          token: ${{ secrets.COVERITY_SCAN_TOKEN }}
```

Comment:
@pbrezina Can you add SSSD repo github secret COVERITY_SCAN_TOKEN from scan.coverity.com WebUI project token (if not done already) ?

Reply:
Done.